src/Sulu/Bundle/CoreBundle/DependencyInjection/SuluCoreExtension.php

```diff
@@ -31,39 +31,6 @@ class SuluCoreExtension extends Extension implements PrependExtensionInterface
      */
     public function prepend(ContainerBuilder $container)
     {
-        // process the configuration of SuluCoreExtension
-        $configs = $container->getExtensionConfig($this->getAlias());
-        $parameterBag = $container->getParameterBag();
-        $configs = $parameterBag->resolveValue($configs);
-        $config = $this->processConfiguration(new Configuration(), $configs);
-
-        if (isset($config['phpcr'])) {
-            $phpcrConfig = $config['phpcr'];
-
-            // TODO: Workaround for issue: https://github.com/doctrine/DoctrinePHPCRBundle/issues/178
-            if (!isset($phpcrConfig['backend']['check_login_on_server'])) {
-                $phpcrConfig['backend']['check_login_on_server'] = false;
```

Comment:
Nope, its for the document manager bundle ..

On Wed, Feb 24, 2016 at 06:05:28AM -0800, Daniel Rotter wrote:
>    In
>    [1]src/Sulu/Bundle/CoreBundle/DependencyInjection/SuluCoreExtension.php:
> 
>  > @@ -31,39 +31,6 @@ class SuluCoreExtension extends Extension implements PrependExtensionInterface
>  >       */
>  >      public function prepend(ContainerBuilder $container)
>  >      {
>  > -        // process the configuration of SuluCoreExtension
>  > -        $configs = $container->getExtensionConfig($this->getAlias());
>  > -        $parameterBag = $container->getParameterBag();
>  > -        $configs = $parameterBag->resolveValue($configs);
>  > -        $config = $this->processConfiguration(new Configuration(), $configs);
>  > -
>  > -        if (isset($config['phpcr'])) {
>  > -            $phpcrConfig = $config['phpcr'];
>  > -
>  > -            // TODO: Workaround for issue: https://github.com/doctrine/DoctrinePHPCRBundle/issues/178
>  > -            if (!isset($phpcrConfig['backend']['check_login_on_server'])) {
>  > -                $phpcrConfig['backend']['check_login_on_server'] = false;
> 
>    I would put that into the prepend method of the SuluContentExtension.
> 
>    —
>    Reply to this email directly or [2]view it on GitHub.
> 
>    Reverse link: [3]unknown
> 
> References
> 
>    Visible links
>    1. https://github.com/sulu-io/sulu/pull/2037#discussion_r53941861
>    2. https://github.com/sulu-io/sulu/pull/2037/files#r53941861
>    3. https://github.com/sulu-io/sulu/pull/2037/files#r53941861